CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing
+
+For documentation changes, you can just edit and make a pull request. Ignore the instructions below.
```

Comment:
Feedback: Don't they also need to search issues?

Response: For now, would rather keep it simple. If it becomes a problem in future, we can revisit it.